ci-operator/step-registry/azure/enable-encryptionathost-day2/azure-enable-encryptionathost-day2-ref.metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+	"path": "azure/enable-encryptionathost-day2/azure-enable-encryptionathost-day2-ref.yaml",
```

Comment:
Maybe we can reuse the existing folder, two options:
1. ci-operator/step-registry/azure/provision/disk-encryption-set/day2/system-managed-key
2. ci-operator/step-registry/ipi/install/post/azure/disk-encryption-set/system-managed-key

WDYT?

Reply:
good suggestion, moved to ci-operator/step-registry/azure/provision/disk-encryption-set/day2/system-managed-key